README.md

```diff
@@ -1,33 +1,65 @@
-# KlaviyoSwift
+# klaviyo-swift-sdk
 
-[![CI Status](https://travis-ci.org/klaviyo/klaviyo-swift-sdk.svg?branch=master)](https://travis-ci.org/klaviyo/klaviyo-swift-sdk)
-[![Swift](https://img.shields.io/badge/Swift-5.6_5.7-orange?style=flat-square)](https://img.shields.io/badge/Swift-5.6_5.7-Orange?style=flat-square)
+![CI status](https://github.com/klaviyo/klaviyo-swift-sdk/actions/workflows/swift.yml/badge.svg)
 [![Swift Package Manager](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)
+![SPM version](https://img.shields.io/github/v/release/klaviyo/klaviyo-swift-sdk)
 [![Version](https://img.shields.io/cocoapods/v/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 [![License](https://img.shields.io/cocoapods/l/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
-[![Platform](https://img.shields.io/cocoapods/p/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
+![Minimum deployment version](https://img.shields.io/badge/minimum_iOS_deployment_target-iOS13-brightgreen)
+
+## Contents
+- [Overview](#overview)
+- [Installation](#installation)
+- [Initialization](#initialization)
+- [Profile Identification](#profile-identification)
+  - [Reset Profile](#reset-profile)
+  - [Anonymous Tracking Notice](#anonymous-tracking-notice)
+- [Event tracking](#event-tracking)
+- [Push Notifications](#push-notifications)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Collecting Push Token](#collecting-push-token)
+    - [Push Tokens And Multiple Profiles](#push-tokens-and-multiple-profiles)
+  - [Request Push Notification Permission](#request-push-notification-permission)
+  - [Receiving Push Notifications](#receiving-push-notifications)
+    - [Tracking opens](#tracking-opens)
+    - [Rich Push Notifications](#rich-push-notifications)
```

Comment:
In the Android readme, it looks like Evan has rich push before tracking opens. Is there a reason you have the order flipped? Should we be consistent?

Reply:
Yeah I should have called this out - I did opens first since that seems more important/required than rich push and also on iOS rich push is additional setup where as on android it's mostly taken care on our SDK. Let me know what you think?

Reply:
My order was more like chronological
Rich Push (displaying) -> Open -> Deep Links

Reply:
For my part, I don't mind the variation in order if the header names are the same.

Reply:
If that order makes the most sense for iOS, let's go with it @ajaysubra .

Reply:
switched it up. thanks!